### ISSUE TITLE
RG Nano USB-C Bluetooth/Wifi Sound Dongle Support

### ISSUE THREAD

**Issue by igeljaeger** [DrUm78/FunKey-OS#12]
**Is your feature request related to a problem? Please describe.**
Lack of sound options outside of the Device's speakers. (USB-C to AUX not working yet etc.)

**Describe the solution you'd like**
I'd love seeing support for bluetooth dongles such as Avantree C81 (pretty much any dongles that work with something like a PS5 and paires independently from the host OS) or/and 2.4GHz Connection Dongles such as used for the SteelSeries Arctis 7+ (maybe easier to implement?).

**Describe alternatives you've considered**
USB-C to AUX

**Additional context**
Basically, I just want to use my Airpods and my Gaming headset with the RG Nano if possible. It certainly is for me on Desktop Linux.

**Comment by DrUm78** [comment]
@Igeljaeger Some Bluetooth dongles are already supported. Here are the ones that have been tested and certified to work on the RG Nano with my latest custom OS:
- Avantree C81
- Creative BT-W2
- Creative BT-W3
- Gulikit Bluetooth Transmitter
- Eppfun AK3040

**Comment by DrUm78** [comment]
@Igeljaeger Avantree C81 is supported with my OS actually:

![image](https://github.com/DrUm78/FunKey-OS/assets/46139740/13412f40-bf72-4028-8f1e-cbac5dc11361)

Discord link: https://discord.com/channels/728036844142592000/1116880154208260126/1136801694773166240

For Wifi dongles, it would need a specific driver and a web browser to use it, which would be unreadable with 240*240 resolution anyway and way too much work, so I close the ticket.
